docs/en/contributor/contributor-tools.md

```diff
@@ -154,8 +169,9 @@ $ mvn test -pl underfs/hdfs -PufsContractTest -DtestHdfsBaseDir=hdfs://ip:port/a
 leverage the Scala shell, as discussed in this
 [blog](http://scala4fun.tumblr.com/post/84791653967/interactivejavacoding).
 
-- The fuse tests are ignored if the libfuse library is missing. To run those tests, please install the correct libraries
-mentioned in [this page]({{ '/en/api/POSIX-API.html' | relativize_url }}#requirements).
+- The fuse tests are ignored if the `libfuse` library is missing.
+To run those tests, please install the correct libraries mentioned in
+[the Alluxio FUSE documentation]({{ '/en/api/FUSE-API.html' | relativize_url }}#requirements).
```

Comment:
I think this has been renamed to Alluxio POSIX API , also with filename to be `/en/api/POSIX-API.html` . please double check

Reply:
Good catch. This went in before I could pull master with the updates. I'll fix it